src/Dapr.Actors/DaprInteractorBuilder.cs

```diff
@@ -0,0 +1,109 @@
+// ------------------------------------------------------------------------
+// Copyright 2022 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+namespace Dapr.Actors
+{
+    using System;
+    using System.Net.Http;
+    using Grpc.Net.Client;
+    using Autogenerated = Dapr.Client.Autogen.Grpc.v1;
+
+    /// <summary>
+    /// Builder for building <see cref="DaprGrpcInteractor"/>
+    /// </summary>
+    public sealed class DaprInteractorBuilder
```

Comment:
Any reason we made this a builder? Also, since it seems generic, if we want to keep it, it should be able to provide an http or grpc one.